components/Main/index.js

```diff
@@ -1,22 +1,155 @@
-import { getLocalStorage } from "../../helpers/localStorage.js";
-import { getProjectFromLocalStorageById } from "../../services/project.js";
+import {
+  getLocalStorage,
+  setLocalStorage,
+} from "../../helpers/localStorage.js";
+import {
+  getProjectById,
+  deleteProject,
+  updateProjects,
+} from "../../services/project.js";
+import { previewProjectsNameOnTheSidebar } from "../Sidebar/index.js";
+import { renderCurrentProject } from "../Sidebar/index.js";
 let proName = document.querySelector(".project-name");
+//  AA!243232
+const threeDotsIcon = document.querySelector(".three-dots-icon");
+const redDeleteIcon = document.querySelector(".red-delete-icon");
+const greenTrueIcon = document.querySelector(".green-true-icon");
```

Comment:
Grouping all DOM queries in one place or using helper functions to get elements can improve readability. It also allows you to change selectors in one place if the HTML structure changes.

```js
// domHelpers.js
// Helper function to get project elements
export const getElement = (selector) => document.querySelector(selector);
```
```js
import { getElement } from "./domHelpers.js";
 

const projectNameElement = getElement(".project-name");
const threeDotsIcon = getElement(".three-dots-icon");
```